package.json

```diff
@@ -7,6 +7,7 @@
   "scripts": {
     "build": "tsc",
     "lint": "ts-standard src/**/*.ts",
+    "lint:fix": "ts-standard src/**/*.ts --fix",
```

Comment:
I don't think it really matters as Actions uses git identifiers, but do you want to bump the version number (`npm version patch`), or zero it out?

Reply:
ah yes, meant to bump it to 2.6, thank you for the reminder!